graph/collection-properties.md

```diff
@@ -0,0 +1,402 @@
+OData services treat collections of complex types differently than they treat collections of entity types. 
+This is due to the nature of entity types being "individually addressable" (they have some key which uniquely identifies them within the collection) while complex types are not individually addressable. 
+The result of this difference is that it is almost always preferable to use entity types for collections rather than complex types; please see the exceptions [here](#exceptions).
+Let's use the following CSDL as an example:
+
+```xml
+<EntityType Name="interestingData">
```

Comment:
Is this intended to be a singleton?  We might want to mention that, as earlier we point out that entity types have keys :)